src/drive/components/LoadMore.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import { Button, Spinner } from 'cozy-ui/react'
+import { translate } from 'cozy-ui/react/I18n'
+import styles from '../styles/table'
+
+const LoadMore = ({ onClick, isLoading, text }) => (
+  <div
+    className={classnames(
+      styles['fil-content-row'],
+      styles['fil-content-row--center']
+    )}
+  >
+    <Button theme="secondary" onClick={onClick}>
+      {isLoading ? <Spinner noMargin /> : text}
+    </Button>
+  </div>
```

Comment:
The Spinner inside the button is not very well centered. I opened a issue on cozy-ui https://github.com/cozy/cozy-ui/issues/234